tests/testthat/test-install.R

```diff
@@ -140,7 +140,7 @@ test_that("various remote styles can be used during install", {
   expect_true(renv_package_version("skeleton") == "1.0.2")
 
   # install from subdir
-  install("kevinushey/subdir:subdir")
+  install("kevinushey/subdir/subdir")
```

Comment:
This is intentional; this is one of the places that `renv` deviates from the `remotes` spec.

Reply:
The reason here being I wanted to resolve the ambiguity in `a/b/c`, since certain remotes might have that mean:

- organization a; user b; repository c,
- user a; repository b; sub-directory c

IIRC this is relevant for GitLab; see https://github.com/rstudio/renv/blob/bab78b78dede06e61c3db3c2163973c4f2747bab/tests/testthat/test-remotes.R#L184-L210

Reply:
PR incoming.